arangodb-net-standard/ViewApi/Models/SearchAliasIndex.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ArangoDBNetStandard.ViewApi.Models
+{
+    public class SearchAliasIndex
+    {
+        /// <summary>
+        /// Possible value for <see cref="Operation"/>
+        /// </summary>
+        public const string SearchAliasIndexAddOperation = "add";
```

Comment:
Should we simplify the name to `AddOperation`? It will be accessed through the class which is named `SearchAliasIndex`.
I think `SearchAliasIndex.AddOperation` reads better than `SearchAliasIndex.SearchAliasIndexAddOperation`?

Same for the others.

Reply:
I've actually created a new static class for operations: `SearchAliasIndexOperation`
I think it's cleaner and more consistent with what we've done in other parts of the driver library.